hw/ip/spi_host/doc/_index.md

```diff
@@ -146,32 +151,117 @@ A preliminary investigation of DTR transfer mode suggests that proper support fo
+  {name: "sd[1]",          wave: "z....................|....2.2.2.2.2.", data: ["d[7]", "d[6]", "d[5]", "d[4]", "..."]},
+  {name: "Segment number", wave: "x2...................|....2.........", data: ['1', '2', '3','4'] },
+  {name: "Segment speed",  wave: "x2...................|....2.........", data: ['Standard', 'Standard'] },
+  {name: "Segment Direction", wave: "x2...................|....2.........", data: ['TX', 'RX', 'None', 'RX'] },
+  ],
+ foot: {text: "Standard SPI Example: Read command with 24-bit address, consisting of one TX and one RX segment"}
+}
+{{< /wavejson >}}
+
+In addition to the TX, RX or Bidirectional modes, many SPI commands require periods where neither the host or device are transmitting data.
+For instance, many flash devices define a Fast Read command in which the host must insert a number of "dummy clocks" between the last address byte and the first data byte from the device.
+These extra cycles are required for operation at higher clock frequencies, to give the address time to propagate through the flash core. 
+A standard-mode Fast Read (with 3 byte addressing) command then requires *three* SPI_HOST command segments:
+- 4 bytes TX Only, one for the instruction code, (i.e., 0xb for Fast Read), and three for the address.
+- 8 dummy clocks 
+- N bytes RX Only for read data response
+
+{{< wavejson >}}
+{signal: [
+  {name: "clk_core_i",     wave: "p....................|.............................."},
+  {name: "sck (CPOL=0)",   wave: "0.1010101010101010101|010101010101010101010101010101"},
+  {name: "csb",            wave: "10...................|.............................."},
+  {name: "sd[0]",          wave: "00.0.0.0.1.0.1.1.2.2.|2.2.x.........................", data: ["a[23]", "a[22]", "a[1]", "a[0]"]},
+  {name: "sd[1]",          wave: "z....................|....z.z.z.z.z.z.z.z.2.2.2.2.2.", data: ["d[7]", "d[6]", "d[5]", "d[4]", "..."]},
+  {name: "Segment number", wave: "x3...................|....4...............5.........", data: ['1', '2', '3'] },
+  {name: "Segment speed",  wave: "x3...................|....4...............5.........", data: ['Standard', 'X', 'Standard'] },
+  {name: "Segment Direction", wave: "x3...................|....4...............5.........", data: ['TX', 'Dummy', 'RX'] },
+  ],
+ foot: {text: "Standard SPI Example: Fast read command (instruction code 0xb) with 24-bit address, consisting of three segments, one TX, 8 dummy clocks and one RX segment"}
+}
+{{< /wavejson >}}
+
+For standard mode-commands, segments simplify the IO process by identifying which bus cycles have useful RX or TX data.
+In such cases it is not strictly necessary to the manage the impedance for the `sd[0]` and `sd[1]` lines.
+For Dual- or Quad-mode commands, however, impedance control is far more important as the impedance for all data lines, `cio_sd[3:0]`, has to dynamically switch between TX and RX segments.
```

Comment:
```suggestion
For Dual- and Quad-mode commands, however, impedance control necessary.
The impedance of all data lines (`cio_sd[3:0]`) has to dynamically switch between TX and RX segments.
```

What do you mean by "dynamically switched"? "dynamic switching" seems already redundant, but it's probably best to slightly rephrase this sentence.

Reply:
This is probably better, I agree.